gcp/compute_engine/cloudrun_serverless_neg_https_lb/variables.tf

```diff
@@ -0,0 +1,76 @@
+variable project {
+  description = "The project to deploy to"
+  type        = string
+}
+
+variable region {
+  description = "Location for serverless_neg and load balancer"
+  type        = string
+  default     = "europe-west-2"
+}
+
+variable serverless_neg_name {
+  description = "Name for serverless network endpoint group"
+  type        = string
+  default     = "serverless-neg"
+}
+
+variable cloudrun_service_name {
+  description = "Cloud run service name"
+  type        = string
+}
+
+variable serverless_https_lb_name {
+  description = "Name for the forwarding rule and prefix for supporting resources"
+  type        = string
+}
+
+variable serverless_https_lb_http_forward {
+  description = "Set to `false` to disable HTTP port 80 forward"
+  type        = bool
+  default     = false
+}
+
+variable serverless_https_lb_ssl {
+  description = "Set to `true` to enable SSL support, requires variable `ssl_certificates` - a list of self_link certs"
+  type        = bool
+  default     = true
+}
+
+variable managed_ssl_certificate_domains {
+  description = "Create Google-managed SSL certificates for specified domains. Requires `ssl` to be set to `true` and `use_ssl_certificates` set to `false`."
+  type        = list(string)
+  default     = []
+}
+
+variable serverless_https_lb_backends {
+  description = "Map backend indices to list of backend maps."
+  type = map(object({
```

Comment:
I'm not clear on why this is a map but only one NEG can be created as part of the module

Reply:
we said we might want to have multiple backends